tests/qunit/index.html

```diff
@@ -59,6 +61,7 @@
 		<script src="wp-admin/js/customize-nav-menus.js"></script>
 		<script src="wp-admin/js/customize-widgets.js"></script>
 		<script src="wp-admin/js/word-count.js"></script>
+		<script src="wp-admin/js/tests.js"></script>
```

Comment:
This name should match the file in wp-admin that is being tested. I believe `updates.js` is what is being tested here.